README.md

```diff
@@ -26,7 +26,10 @@ jobs:
           S3_PATH: "${S3_PATH}"
           AWS_ACCESS_KEY_ID: ${{ secrets.DOC_WRITER_AWS_ACCESS_KEY_ID }}
           AWS_SECRET_ACCESS_KEY: ${{ secrets.DOC_WRITER_AWS_SECRET_ACCESS_KEY }}
-
+          CHANGELOG: true # generate a changelog file out of Github release notes
+          CHANGELOG_STRIP_PATTERN: https?:\/\/[^ ]+ # optional strip pattern to run on the generated changelog
```

Comment:
I'm using this to strip links like `https://github.com/play-co/gcinstant/compare/v12.16.1...v12.17.0` which mess up the table of contents in GCInstant's generated changelog.